examples/taco/nextjs/src/app/page.tsx

```diff
@@ -73,16 +73,15 @@ function App() {
 
     console.log('Encrypting message...');
     const hasPositiveBalance = new conditions.RpcCondition({
-      conditionType: 'rpc',
-      chain: 5,
+      chain: 80001,
```

Comment:
@piotr-roslaniec / @KPrasch Does changing the chain from 5 -> 80001 affect any default behaviour / expectations of the demo...?

Reply:
It doesn't but it makes the example more consistent (everything is happening on the same chain).

Reply:
The tapir nodes are not evaluating chain 5 for conditions, but 80001 is used for both testnet and devnet.  I figure this gives the examples/demos the best shot at working "out of the box" regardless of which testnet you use.